renderdoc/driver/vulkan/vk_postvs.cpp

```diff
@@ -659,6 +659,38 @@ static void ConvertToMeshOutputCompute(const ShaderReflection &refl, const SPIRV
   for(SPIRVIterator end = editor.EndEntries(); it < end; ++it)
     editor.Remove(it);
 
+  // Strip away any execution modes from the original shaders
+  for (SPIRVIterator it = editor.BeginEntries(), end = editor.BeginDebug(); it < end; ++it) {
+     if (it.opcode() == spv::OpExecutionMode) {
+        auto modeEntryID = static_cast<SPIRVId>(it.word(1));
```

Comment:
Avoid `static_cast` and use of `auto` for simple types please,

```suggestion
        SPIRVId modeEntryID = SPIRVId(it.word(1));
```

Same below for `execMode`.